easybuild/easyconfigs/i/IRkernel/IRkernel-1.1.1-foss-2020a-R-4.0.0-Python-3.8.2.eb

```diff
@@ -0,0 +1,68 @@
+easyblock = 'Bundle'
+
+name = 'IRkernel'
+version = '1.1.1'
+versionsuffix = '-R-%(rver)s-Python-%(pyver)s'
+
+homepage = 'https://irkernel.github.io'
+description = """The R kernel for the 'Jupyter' environment executes R code
+ which the front-end (Jupyter Notebook or other front-ends) submits to the
+ kernel via the network."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+dependencies = [
+    ('R', '4.0.0'),
+    ('Python', '3.8.2'),
+    ('IPython', '7.15.0', '-Python-%(pyver)s'),
+    ('ZeroMQ', '4.3.2'),
+]
+
+exts_defaultclass = 'RPackage'
+exts_filter = ("R -q --no-save", "library(%(ext_name)s)")
+
+exts_default_options = {
+    'source_urls': [
+        'https://cran.r-project.org/src/contrib/',
+        'https://cran.rstudio.com/src/contrib/',
+        'https://cran.r-project.org/src/contrib/Archive/%(name)s/',
+    ],
+    'source_tmpl': '%(name)s_%(version)s.tar.gz',
+}
+
+exts_list = [
+    ('repr', '1.1.0', {
```

Comment:
Maybe there is a reason not to update, but there seem to be newer versions available.